cloudbuild/run-presubmit-on-k8s.sh

```diff
@@ -14,6 +14,7 @@ kubectl run "${POD_NAME}" \
   --image="${IMAGE}" \
   --pod-running-timeout=15m \
   --restart=Never \
+  --limits="cpu=500m,ephemeral-storage=2Gi,memory=2Gi" \
```

Comment:
This is not supported in new k8s version - we need use future-proof config, or remove it - otherwise it will break in the future.

Reply:
Removed